4.0-MIGRATION-GUIDE.md

```diff
@@ -19,6 +37,50 @@ Non-breaking:
   - [downlevel-dts](https://github.com/sandersn/downlevel-dts) was used to resolve breaking changes
     in `typescript@3.7`, so older versions of typescript are still supported. However, it is
     recommended to use v3.8.
+- Many of our dependencies have been updates to address a low level vulnerability. This shouldn't
+  affect your day to day usage of Canvas Kit.
+
+## Theming Changes
+
+We have promoted all of the theming functionality out of Canvas Kit Labs.
+[It now lives in `@workday/canvas-kit-react-common](https://github.com/Workday/canvas-kit/tree/prerelease/v4/modules/common/react/lib/theming).
+We've also made some stability improvements.
```

Comment:
Since `CanvasProvider` isn't exactly part of theme, is it worth noting that we also moved it to `common` ?